Makefile

```diff
@@ -144,6 +144,24 @@ dep-update:
 	@echo Updating dependencies and storing in vendor directory
 	@glide update --strip-vendor
 
+#
+# Swagger Documentation
+#
+swagger-install:
+	go get -u github.com/go-swagger/go-swagger/cmd/swagger
+
+swagger-validate:
+	@swagger validate ./swagger.json
+	@if [ "$$?" != "0" ]; then \
+        exit 1; \
+    fi
```

Comment:
nit but the `if ...` part is unnecessary since `make` already returns the exit code in case the swagger command fails.